src/main/java/core/basesyntax/Car.java

```diff
@@ -1,5 +1,26 @@
 package core.basesyntax;
 
-public class Car {
+public class Car implements Cloneable {
+    private String name;
+    private String colour;
+    private String type;
+    private int maxSpeed;
+    private int acceleration;
+    private Engine engine;
 
+    public Car(String name, String colour, String type,
+               int maxSpeed, int fuelType, Engine engine) {
+        this.name = name;
+        this.colour = colour;
+        this.type = type;
+        this.maxSpeed = maxSpeed;
+        this.acceleration = acceleration;
+        this.engine = engine;
+    }
+
+    @Override
+
```

Comment:
there shouldn't be any empty lines between annotation and method

Reply:
done